docs/matrix.md

```diff
@@ -154,11 +160,12 @@ spec:
 ```
 
 ### Include
+
```

Comment:
Is this feature still in "preview mode"? TBH I'm not sure what "preview mode" means-- can we describe it as an alpha feature instead, and add to the alpha features table?

Reply:
I think we can remove preview mode now.